gui.go

```diff
@@ -130,7 +139,7 @@ func (gui *GUI) NewSliderWannabe(slider SliderWannabe) {
 	gui.NewButton(minusButton)
 
 	plusButton := Button{
-		position:     pixel.V(190, slider.y),
+		position:     pixel.V(slider.canvasWidth-60-10, slider.y),
```

Comment:
What is `-60-10`? If it is possible initializing constants would be helpful to explain it or at least providing some explanation in comment would be helpful. I realized that I am little bit confised, it is hard to understand our code without any explanations.